vtex-apps/return-app
# V3 status timeline

#### What problem is this solving?
This PR brings the status timeline and comments to the return request details page.

#### How to test it?



In this [workspace](https://rmav3timelinestatus--powerplanet.myvtex.com/admin/), it's possible to see the component. It's worth visiting a request not closed yet and another one closed with status denied. One can add comments, that should be displayed right away in the component.

#### Screenshots or example usage:
_Request not closed_
<img width="933" alt="Screen Shot 2022-06-17 at 2 29 02 PM" src="https://user-images.githubusercontent.com/38737958/174298414-00d197dd-6a3f-46d4-b409-e1d796554f7c.png">

_Request denied_
<img width="675" alt="Screen Shot 2022-06-17 at 2 29 37 PM" src="https://user-images.githubusercontent.com/38737958/174298435-47382851-2bde-4a30-83ac-2ceb8372fe19.png">
